### PR TITLE
Add parser for Mysql-server's log file

### PR DESCRIPTION
**Pull request by JoySnow**
Signed-off-by: XiaoXue Wang <xiaoxwan@redhat.com>

### CODE REVIEW

**Comment by bfahr** [comment]
@JoySnow would it make sense to add this parser to the existing Maria DB log file parser since the documentation is the same?

**Comment by JoySnow** [comment]
@bfahr  The documentation is not the same. I'd like to keep this parser in a separate file. Or the `mariadb_log.py` need change the file name to include both parsers.

File: insights/parsers/mysql_log.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    2018-03-13T06:37:41.080591Z 1 [Warning] root@localhost is created with an empty password ! Please consider switching off the --initialize-insecure option.
+    md5_dgst.c(80): OpenSSL internal error, assertion failed: Digest MD5 forbidden in FIPS mode!
+    06:37:41 UTC - mysqld got signal 6 ;
+    2018-03-13T07:43:31.450772Z 0 [Note] Event Scheduler: Loaded 0 events
+    2018-03-13T07:43:31.450988Z 0 [Note] /opt/rh/rh-mysql57/root/usr/libexec/mysqld: ready for connections.
+    Version: '5.7.16'  socket: '/var/lib/mysql/mysql.sock'  port: 3306  MySQL Community Server (GPL)
+
+Examples:
+
+    >>> my = shared[MysqlLog]
```
Comment (xiangce) [inline insights/parsers/mysql_log.py]: bye-bye `shared`

File: insights/parsers/mysql_log.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
+"""
+MysqlLog - File ``/var/log/mysqld.log``
+=======================================
+
+Module for parsing the log file for Mysql, including ``/var/log/mysqld.log`` and
+``/var/opt/rh/rh-mysql*/log/mysql/mysqld.log``.
+
+Note:
```
Comment (xiangce) [inline insights/parsers/mysql_log.py]: Change to `.. note::`, it will look better in HTML document.

**Pull request merged**